test/built-ins/Iterator/prototype/toArray/iterator-already-exhausted.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2023 Michael Ficarra. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-iteratorprototype.toArray
+description: >
+  Iterator.prototype.toArray returns undefined when the iterator has already been exhausted
```

Comment:
```suggestion
  Iterator.prototype.toArray returns an empty array when the iterator has already been exhausted
```

Also the original iterator is empty, so you'd get `[]` either way; it's not a very informative test.